<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
    <item>
     <widget class="QCustomPlot" name="plotReal" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="plotFFT" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>debug</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlots">
    <property name="title">
     <string>Plots</string>
    </property>
    <addaction name="actionShow_real_data"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEqualizer"/>
    <addaction name="actionNormalization"/>
   </widget>
   <addaction name="menuDebug"/>
   <addaction name="menuPlots"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_real_data"/>
   <addaction name="separator"/>
   <addaction name="actionEqualizer"/>
   <addaction name="actionNormalization"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShow_real_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show real data</string>
   </property>
  </action>
  <action name="actionEqualizer">
   <property name="text">
    <string>Equalizer</string>
   </property>
  </action>
  <action name="actionNormalization">
   <property name="text">
    <string>Compression</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
